gg.minion.Minion.yml

```diff
@@ -0,0 +1,75 @@
+app-id: gg.minion.Minion
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+command: minion.sh
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+finish-args:
+  - --env=PATH=/app/java/jre/bin:/app/bin
+  - --socket=x11
+  - --share=network
+  - --filesystem=home
```

Comment:
Have you checked if `--persist=.minion` helps here?

Reply:
That seems to have fixed the issue I was having, thanks!